src/firefly/js/drawingLayers/Catalog.js

```diff
@@ -142,11 +153,29 @@ function getLayerChanges(drawLayer, action) {
     var dd= Object.assign({},drawLayer.drawData);
     dd[DataTypes.HIGHLIGHT_DATA]= null;
     if (changes.tableData) dd[DataTypes.DATA]= null;
+    if (changes.selectInfo) dd[DataTypes.SELECTED_IDXS]= null;
     return Object.assign({}, changes,{drawData:dd});
 }
 
 
-const findColIdx= (columns,colId) => columns.findIndex( (c) => c.name===colId);
+
+function getDrawData(dataType, plotId, drawLayer, action, lastDataRet) {
+
+    const{tableData, columns}= drawLayer;
+    switch (dataType) {
+        case DataTypes.DATA:
+            return isEmpty(lastDataRet) ? computeDrawLayer(tableData, columns) : lastDataRet;
+        case DataTypes.HIGHLIGHT_DATA:
```

Comment:
What happens if you want both tables to go to that spot? I am going to go ahead and keep it as it is for this ticket and we can talk about the right way to do it.  I have mixed feelings so I made it affect all the visible layers for all the functions: highlight, select, filter.  Table that are uncheck are not affected.